packages/components/pie-notification/test/component/pie-notification.spec.ts

```diff
@@ -1,20 +1,237 @@
 
 import { test, expect } from '@sand4rt/experimental-ct-web';
 import { PieNotification, NotificationProps } from '../../src/index.ts';
+import { variants, headingLevels } from '../../src/defs.ts';
 
-const componentSelector = '[data-test-id="pie-notification"]';
+const rootSelector = 'pie-notification';
+const componentSelector = `[data-test-id="${rootSelector}"]`;
+const iconCloseSelector = `[data-test-id="${rootSelector}-icon-close"]`;
+const slottedIconSelector = `[data-test-id="${rootSelector}-icon-slotted"]`;
+const headingIconInfoSelector = `[data-test-id="${rootSelector}-heading-icon-info"]`;
+const headingIconSuccessSelector = `[data-test-id="${rootSelector}-heading-icon-success"]`;
+const headingIconWarningSelector = `[data-test-id="${rootSelector}-heading-icon-warning"]`;
+const headingIconErrorSelector = `[data-test-id="${rootSelector}-heading-icon-error"]`;
+const headerSelector = `[data-test-id="${rootSelector}-header"]`;
+const headingSelector = `[data-test-id="${rootSelector}-heading"]`;
+
+const defaultProps: NotificationProps = {
+    variant: 'neutral',
+    compact: false,
+    heading: 'Title',
+    headingLevel: 'h2',
+    hideIcon: false,
+    isOpen: true,
+};
+
+const slotContent = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sit amet tincidunt est, vitae vulputate turpis. Cras pretium venenatis elementum. Duis tristique neque non varius tempor. In hac habitasse platea dictumst. Aenean accumsan vehicula urna. Cras fringilla sed ipsum nec dignissim. Aliquam sit amet ullamcorper ligula.';
+const mockSlottedIcon = `<div slot="icon" data-test-id="${rootSelector}-icon-slotted">Mocked Icon Slot</div>`;
 
 test.describe('PieNotification - Component tests', () => {
-    test('should render successfully', async ({ mount, page }) => {
+    test('should render successfully with default props', async ({ mount, page }) => {
```

Comment:
please could this file follow the structure of components such as pie-input? It has a nice structure for props tests, integration tests etc

Reply:
I agree! I just pushed and overhaul in the unit tests following the same structure. Thank you